src/components/calcite-demo-nav/calcite-demo-nav.tsx

```diff
@@ -144,7 +144,9 @@ export class CalciteDemoNav {
   render() {
     return (
       <Host onBlur={this.blurHandler.bind(this)}>
-        <button class="hamburger" onClick={this.hamburgerClickHandler.bind(this)}></button>
+        <button class="hamburger" onClick={this.hamburgerClickHandler.bind(this)}>
+          <CalciteIcon size="32" path={hamburger32} title="Home" />
```

Comment:
should we have a `textHome` prop for the title?

Reply:
I don't think so - this isn't intended to be used outside of our demo section so this doesn't need to be dynamic.